README.md

```diff
@@ -25,12 +25,12 @@ from cloudevents.sdk.event import v01
 
 event = (
     v01.Event().
-    WithContentType("application/json").
-    WithData('{"name":"john"}').
-    WithEventID("my-id").
-    WithSource("from-galaxy-far-far-away").
-    WithEventTime("tomorrow").
-    WithEventType("cloudevent.greet.you")
+    SetContentType("application/json").
+    SetData('{"name":"john"}').
+    SetEventID("my-id").
```

Comment:
any reason this isn't just `SetID` to align with the property name change? Same for Time and Type below.

Reply:
That’s what’s in SDK spec. Verbose API is a better option here, less things seem to be ambiguous.